src/communication/graphdatahandler.cpp

```diff
@@ -66,7 +66,7 @@ qint32 GraphDataHandler::expressionErrorPos(qint32 exprIdx) const
     return _valueParsers[exprIdx].errorPos();
 }
 
-void GraphDataHandler::handleRegisterData(QList<Result<qint64> > results)
+void GraphDataHandler::handleRegisterData(QList<Result<double> > results)
```

Comment:
It seems that the double data type is capable of handling all integers up to `2⁵³ - 1`. I think this will be enough. Personally I would be happy if the full range of an uint32_t is supported.

Source: https://stackoverflow.com/questions/3793838/which-is-the-first-integer-that-an-ieee-754-float-is-incapable-of-representing-e

Reply:
Great!